taxi/models.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.conf import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+    country = models.CharField(max_length=100)
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=100, unique=True)
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=100)
+    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE)
+    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL,
+                                     related_name="drivers")
```

Comment:
The `related_name` should be the name of the model in plural form. In this case, it should be `cars` instead of `drivers`. This will allow you to access all cars related to a driver using `driver.cars`.